### PR TITLE
CODE RUB: Fix One Liner 3

### PR DESCRIPTION
**Pull request by devanteconnor**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs [hunk 1]
```diff
@@ -34,10 +34,10 @@ public AssignmentAttachmentServiceTests()
         }
 
         private static AssignmentAttachment CreateRandomAssignmentAttachment() =>
-            CreateAssignmentAttachmentFiller(dates: GetRandomDateTime()).Create();
+            CreateAssignmentAttachmentFiller(DateTimeOffset.UtcNow).Create();
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs:37]: Revert these changes

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Fix your PR title

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs (hunk 1 above)
Comment (devanteconnor) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs:37]: Same for this I didnt make any changes for Assignment.

**Comment by ShriHumrudha** [comment]
Closing the PR as it is beyond the 3-day window.
